docs/build/js-client/10-marine-js.md

```diff
@@ -2,11 +2,36 @@
 
 > This section is experimental as the API is not stable and probably will change.
 
+## What is Marine JS
+
+### Intro
+When JS client starts up, it starts with initialization Marine JS runtime.
+The runtime hosts wasm services and even allows you to register your own wasm services.
+For example, AquaVM service for processing particles and Air code resides in Marine JS.
+That's why Marine JS is a foundation of JS client. 
+
+### Use cases
+
+> To register your own service, you would usually use JS-client js services as they are simpler to write and implement.
+
+Usually you don't need to interact with Marine JS directly. 
+AquaVM service being registered at JS client startup phase and ready to process your compiled Aqua code.
+
+So if your needs include any of the following:
+- Cross-platform service which works both on Nox peer and JS client peer
+- Heavy computations, CPU intensive tasks
+- Would prefer to write service in Rust or other language with compilation WASM
+
+In that case, writing your service as WASM more suitable than doing so in JavaScript.
+
```

Comment:
i understand that you may want to reuse or otherwise benefit from marine services but i still don't understand why to run the js peer.

Reply:
Wdym by "run the js peer" ?

Reply:
I think the question is "why use js-peer to run marine when you can use nox to run marine services there". Not sure if it's correct. And also not sure what the correct answer here would be